scripts/run_mypy_checks.py

```diff
@@ -39,7 +42,13 @@
     # ignored while type checking, because these files are only
     # used as resources for the tests.
     'core/tests/build_sources/',
-    'core/tests/data/'
+    'core/tests/data/',
+    'core/jobs/io/gcs_io.py',
+    'core/jobs/batch_jobs/translation_migration_jobs.py',
+    'core/jobs/batch_jobs/suggestion_stats_computation_jobs.py',
+    'core/jobs/batch_jobs/suggestion_migration_jobs.py',
+    'core/jobs/batch_jobs/opportunity_management_jobs.py',
+    'core/jobs/batch_jobs/exp_migration_jobs.py'
```

Comment:
Only one comment! Why these files are excluded from strict typing ?

Reply:
IDK why there were failing under Docker setup, so I added them in the ignore list as I dont have changed anything in any of these files. my suspect was that under python setup, somehow they were getting skipped or ignored, but docker proving a robust env in the containers, the are getting checked and throwing errors under docker setup.

Reply:
these were the error logs ---
```
core/jobs/batch_jobs/translation_migration_jobs.py needs an associated backend test file.
core/jobs/batch_jobs/suggestion_stats_computation_jobs.py needs an associated backend test file.
core/jobs/batch_jobs/suggestion_migration_jobs.py needs an associated backend test file.
.
.
.
```

Reply:
@sahiljoster32 FYI, I need to exclude some more scripts from the backend associated tests. please [refer](https://docs.google.com/document/d/1pVv6m_wUcvx4CYg5C_2pBkgA1Pklj6esLaBWqssSGwI/edit?disco=AAAA7aBvTqc), I was getting the similar issue under Docker

Reply:
yes, I will check this soon and update it here

Reply:
reverting these changes as not required. they are passing on my local as well as I checked on CI under Docker setup on my other PR